.gitignore

```diff
@@ -1,5 +1,4 @@
 /sequins
-/sequins-dump
```

Comment:
I generally do too, but I didn't in this case because the intermediate state wouldn't have compiled. (It would've been possible to make that happen, but it didn't seem worth the effort)